content/docs/tasks/traffic-management/circuit-breaking/index.md

```diff
@@ -40,6 +40,36 @@ configuration by intentionally "tripping" the circuit breaker.
 1.  Create a [destination rule](/docs/reference/config/istio.networking.v1alpha3/#DestinationRule) to apply circuit breaking settings
 when calling the `httpbin` service:
 
+    If you have enabled default mutual TLS authentication, run this command:
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -f -
+    apiVersion: networking.istio.io/v1alpha3
+    kind: DestinationRule
+    metadata:
+      name: httpbin
+    spec:
+      host: httpbin
+      trafficPolicy:
+        connectionPool:
+          tcp:
+            maxConnections: 1
+          http:
+            http1MaxPendingRequests: 1
+            maxRequestsPerConnection: 1
+        outlierDetection:
+          consecutiveErrors: 1
+          interval: 1s
+          baseEjectionTime: 3m
+          maxEjectionPercent: 100
+        tls:
+          mode: ISTIO_MUTUAL
+
+    EOF
+    {{< /text >}}
+
+    otherwise, run this command:
+
```

Comment:
The mirroring task had this same issue, but instead of duplicating the whole `DestinationRule` we just added a note.

https://preliminary.istio.io/docs/tasks/traffic-management/mirroring/#creating-a-default-routing-policy

I think we should do the same thing here, but the note should be a little more clear. I would suggest the following:

> Note: If you installed/configured Istio with mutual TLS Authentication enabled, you must add a TLS traffic policy `mode: ISTIO_MUTUAL` to the `DestinationRule` before applying it. Otherwise requests will generate 503 errors as described [here](https://istio.io/help/ops/traffic-management/deploy-guidelines/#503-errors-after-setting-destination-rule).

Reply:
I thought about adding a note too, but for the novice user, it might be confusing to add that piece of config in the main DestinationRule object. It breaks the copy&paste model of following a tutorial. A better alternative (IMO) which has less clutter would be having tabs, for example, like [this](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer).

Do we have support for tabs?

Reply:
Having those tabs would be nice @geeknoid

Reply:
We don't support for these kinds of tabs at this point, but there is definitely a need. I should be able to put something like that together reasonably quickly though. I might even be able to steal some logic from the k8s doc repo in fact :-)

Reply:
So, can we go ahead with those two slightly different DestinationRules objects and use tabs when they are available? Or do you still prefer having the note?

Reply:
I think we should fix both tasks (circuit-breaker and mirroring) the same way and I think we should have the note that I suggested in the doc whether or not we have tabs. Even if we have tabs, I think that the note is important to help people figure out which tab to use, they might not even be sure if they have mTLS enabled or not. The note I suggested also has a link to the doc that helps users understand what happened if they get the 503s.

So, my proposal is that we should add the note for now (both tasks) and wait until we have tabs to add the second yaml. Adding it without tabs is too much noise IMO and I think most beginners start without mTLS, so it's not too bad for now. As soon as we have tabs, I agree it would be good to put both yamls in, but like I said, leave the note there as well.